content/membership.inc

```diff
@@ -39,77 +17,10 @@
 
     <p>Being a member of the Buggy Alumni Association is an annual commitment, and our "year" starts and ends at Raceday. All gifts received before and during carnival make you a member of the BAA team for that race-year. For example, gifts received May 2011-April 2012 make you a member of the 2012 team.</p>
 
-    <a target="_blank" onclick="_gaq.push(['_trackEvent','Outbound','Membership',this.href]);" href="https://securelb.imodules.com/s/1410/giving/form.aspx?sid=1410&gid=1&pgid=382&cid=990&appealcode=A3291&dids=98" class="btn btn-primary mb-3">Join (or Renew) Now</a>
+    <a target="_blank" onclick="_gaq.push(['_trackEvent','Outbound','Membership',this.href]);" href="https://cmu.secure.force.com/pmtx/giftselect?id=a41f4000000eIQ0" class="btn btn-primary mb-3">Join (or Renew) Now</a>
```

Comment:
Hmm.. `_gaq` = "Google Analytics Queue"? Wonder if we can find where that's going / how to get in